### DISCUSSION TITLE
Very slow conversion

### DISCUSSION THREAD

**Discussion by jglazer63** [RandomEngy/VidCoder#947] [General]
Does VidCoder use any hardware acceleration?  It takes about an hour to encode an hour of video where others take about 5 minutes.  Perhaps I am doing something wrong.  I specifically want to encode my videos to H.265 (for space) with only English Forced subtitles.  It just seems to be exceptionally slow and can only do one encoding at a time.

Any suggestions?

**Comment by RandomEngy** [comment]
You use hardware encoding by picking a different encoder from the list on the Video Encoding tab. If it says "QSV", "NVenc" or "VCE" then it's hardware-accelerated. Also sometimes a filter like resize or cropping can be a bottleneck on multi-core systems, so it might be worth trying to turn all filters off and see if that changes the speed.

**Reply by jglazer63** [reply]
> You use hardware encoding by picking a different encoder from the list on the Video Encoding tab. If it says "QSV", "NVenc" or "VCE" then it's hardware-accelerated. Also sometimes a filter like resize or cropping can be a bottleneck on multi-core systems, so it might be worth trying to turn all filters off and see if that changes the speed.

This is great and exactly what I wanted but I changed it from the Default H.265 720p to using NVenc and that, being the only change, more than doubled the size of the output file.  Why?

**Reply by RandomEngy** [reply]
The quality target levels and encoder efficiency isn't the same across the two encoders. Also NVEnc is meant for low CPU-power real-time streaming so it's not going to quite as efficient in encoding to any given quality target level.
